apps/juror-digital/identity/juror-digital-azure-identity.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: juror-digital
+spec:
+  type: 0
+  resourceID: replace
```

Comment:
where's the dev identity for this?

Reply:
There wasn't one in the flux v1 setup but I've added it there

Reply:
Must’ve been somewhere